.github/workflows/lint-build-test.yml

```diff
@@ -0,0 +1,42 @@
+name: lint-build-test
+# build on every PR and commit to master
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: install dependencies
+      run: python -m pip install --upgrade pip
+    - name: lint
+      run: |
+        pip install -q https://github.com/qiime2/q2lint/archive/master.zip
+        q2lint
+        pip install -q flake8
+        flake8
+
+  build-and-test:
+    needs: lint
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    # for versioneer
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
```

Comment:
@thermokarst, I'm trying to figure out why this line is failing. Based on a little light reading, I'm guessing it's trying to grab the latest version tag, which I'm assuming versioneer is using to apply a new version patch based on version and SHA? 

Am I overlooking some configuration thing git needs for this? Or a config file Actions is expecting? (I didn't see anything about the latter in the docs, but I could easily have missed something.) 

When I run this line manually on my local, it returns nothing, even though `git tag -l` returns these: 
```
2019.4.0.dev0
2019.7.0.dev0
```

Both linux and osx builds appear to be failing here. 
![image](https://user-images.githubusercontent.com/39198770/82503519-4cb2e180-9aae-11ea-83c7-bc4b3dedca33.png)

With this section removed, the rest of the job fails, and I suspect it's struggling with undefined/malformed version numbers.

Reply:
> I'm guessing it's trying to grab the latest version tag, which I'm assuming versioneer is using to apply a new version patch based on version and SHA?

Yes indeed!

> Am I overlooking some configuration thing git needs for this?

Yes indeed! Your repo has no tags pushed up to it: https://github.com/ChrisKeefe/q2-diversity-lib/tags

Reply:
Interesting! So this wouldn't be failing if this were a PR on qiime2/q2-diversity-lib...

As far as best practices go, should I be manually pushing tags to my fork? or using something like `git push --follow-tags`, possibly as a config option?

Reply:
I think this will come down to your personal taste, but yeah, syncing tags with your fork probably wouldnt be the worst idea, and setting up a config in your global gitconfig might make that easier to manage.

In the meantime, push up a tag on master that looks like this: https://github.com/caporaso-lab/q2-phylogenomics/releases/tag/2020.5.0.dev0 (although I would call it 2020.6.0.dev0, since we aren't doing a 2020.5 release anymore).

Reply:
FYI: when using a `2020.6.0.dev0` tag, I get the following failure, so I swapped back to 2020.5. Seems to have resolved that issue.
```
conda.exceptions.ResolvePackageNotFound: 
  - q2-types=2020.6
  - qiime2=2020.6
```

Reply:
This is what I was talking about over here ([link](https://github.com/qiime2/q2-diversity-lib/pull/9#issuecomment-631640161)) when I said:

> I'm not 100% this'll work at the moment

The reason it is failing on 2020.6 is because the action doesn't know about 2020.6 packages (yet), just need to update this line:

https://github.com/qiime2/action-library-packaging/blob/alpha/src/main.ts#L15

You will need to set the tag to 2020.6 once you're done debugging.

Reply:
Scratch that - you'll need to update to 2020.6 now - I updated the GitHub action.